cfpb/cfgov-refresh
# External link manager front end updates

Front end updates to the external link manager


## Changes

- Overrides the base Wagtail page table template to limit the action buttons and open most links in a new tab (parent page link still opens in same page)
- Nav icon for external link manager
- Layout/style updates to external link manager, including inline search form

## Testing

1. Check out branch
2. Navigate to [external link manager](http://localhost:8000/admin/external-links/) and verify that page matches screenshot
3. Run a link search and verify that results match screenshot

## Screenshots

<img width="1022" alt="screen shot 2018-10-17 at 2 39 50 pm" src="https://user-images.githubusercontent.com/778171/47117898-8a88bd80-d21a-11e8-8171-cd5eb326c75b.png">

## Checklist
